--- build/2-darby.xhtml	2018-03-01 15:42:54.085002564 +0100
+++ build/3-darby.xhtml	2018-03-01 15:42:58.822002498 +0100
@@ -848,7 +848,7 @@
     <p class="Usuel">24 sauf seulement ce qu'ont mangé les jeunes gens, et la part des hommes qui sont allés avec moi,
     Aner, Eshcol et Mamré&#160;: eux, ils prendront leur part.</p>
     <p class="Note">— v. 18*&#160;: roi de justice. — v. 18**&#160;: <i>hébreu</i>&#160;: El, le Fort, <i>distingué
-    toujours</i> d'Élohim (Dieu), <i>et</i> d'Éloah (#Dieu), <i>par</i> <i>l'astérisque* précédant le mot de</i>
+    toujours</i> d'Élohim (Dieu), <i>et</i> d'Éloah (#Dieu), <i>par</i> <i>l'astérisque∗ précédant le mot de</i>
     Dieu&#160;; <i>comparer</i> Genèse 1:1 <i>et</i> Deut. 32:15. — v. 18***&#160;: Très-haut, <i>en hébreu</i>&#160;:
     Élion. — v. 21&#160;: <i>litt.</i>: âmes.</p>
     <p class="Clustermoyen"><a name="at01_15" id="at01_15"></a>*</p>
@@ -3228,7 +3228,7 @@
     <p class="Note">— v. 22&#160;: <i>hébreu</i>&#160;: Sichem.</p>
     <p class="Clustermoyen">*</p>
     <p class="Chapitre"><a name="at01_49" id="at01_49"></a>Chapitre 49</p>
-    <p class="Posie">1 Et Jacob appela ses fils, et dit&#160;: Assemblez-vous, et je vous ferai savoir ce qui vous
+    <p class="Usuel">1 Et Jacob appela ses fils, et dit&#160;: Assemblez-vous, et je vous ferai savoir ce qui vous
     arrivera à la fin des jours.</p>
     <p class="Posie">2 Réunissez-vous, et écoutez, fils de Jacob&#160;; écoutez Israël, votre père.</p>
     <p class="Clustersecondaire">*</p>
@@ -29131,7 +29131,7 @@
     <p class="Usuel">11 Et tu fendis la mer devant eux, et ils passèrent à sec par le milieu de la mer&#160;; et ceux
     qui les poursuivaient, tu les jetas dans les abîmes, comme une pierre dans les eaux puissantes.</p>
     <p class="Note">— v. 6, 7&#160;: Tu [es] le Même, <i>ou</i>&#160;: Tu [es] Celui [qui est]&#160;: <i>cette
-    expression devient un vrai nom de</i> Dieu&#160;; <i>comparer</i> Deut. 32:39&#160;; Ps. 102:27&#160;; És.
+    expression devient un vrai nom de</i> Dieu&#160;; <i>comparer</i> Deut. 32:39&#160;; Ps. 102:27&#160;; Ésaïe
     41:4.</p>
     <p class="Usuel">12 Et tu les conduisis de jour par une colonne de nuée, et de nuit par une colonne de feu, afin
     d'éclairer pour eux le chemin dans lequel ils devaient marcher.</p>
@@ -30168,7 +30168,7 @@
     <p class="Posie">20 Du matin au soir, ils sont frappés&#160;; ils périssent pour toujours sans qu'on y fasse
     attention.</p>
     <p class="Posie">21 Leurs cordes* ne leur sont-elles pas arrachées&#160;? Ils meurent, et sans sagesse.</p>
-    <p class="Note">— v. 21&#160;: les cordes qui retiennent une tente&#160;; <i>comparer</i> És. 33:20.</p>
+    <p class="Note">— v. 21&#160;: les cordes qui retiennent une tente&#160;; <i>comparer</i> Ésaïe 33:20.</p>
     <p class="Chapitre"><a name="at18_05" id="at18_05"></a>Chapitre 5</p>
     <p class="Posie">1 Crie donc&#160;! Y a-t-il quelqu'un qui te réponde&#160;? Et vers lequel des saints te
     tourneras-tu&#160;?</p>
@@ -31882,6 +31882,7 @@
     4</a> (Ps. 90 à 106), <a href="#at19_5">Livre 5</a> (Ps. 107 à 150)</p>
     <p class="Usuel">&#160;</p>
     <p class="Clustersuprieur"><a name="at19_1" id="at19_1"></a>LIVRE PREMIER</p>
+    <!--
     <p class="Usuel" align="center"><a href="#at19_1_01">1</a>, <a href="#at19_1_02">2</a>, <a href="#at19_1_03">3</a>,
     <a href="#at19_1_04">4</a>, <a href="#at19_1_05">5</a>, <a href="#at19_1_06">6</a>, <a href="#at19_1_07">7</a>,
     <a href="#at19_1_08">8</a>, <a href="#at19_1_09">9</a>, <a href="#at19_1_10">10</a>, <a href="#at19_1_11">11</a>,
@@ -31893,6 +31894,7 @@
     <a href="#at19_1_32">32</a>, <a href="#at19_1_33">33</a>, <a href="#at19_1_34">34</a>, <a href="#at19_1_35">35</a>,
     <a href="#at19_1_36">36</a>, <a href="#at19_1_37">37</a>, <a href="#at19_1_38">38</a>, <a href="#at19_1_39">39</a>,
     <a href="#at19_1_40">40</a>, <a href="#at19_1_41">41</a></p>
+    -->
     <p class="Usuel">&#160;</p>
     <p class="Chapitre"><a name="at19_1_01" id="at19_1_01"></a>PSAUME 1</p>
     <p class="Usuel">1 Bienheureux l'homme qui ne marche pas dans le conseil des méchants, et ne se tient pas dans le
@@ -32589,8 +32591,8 @@
     entrera.</p>
     <p class="Usuel">10 Qui est-il, ce roi de gloire&#160;? L'Éternel des armées, lui, est le roi de gloire. Sélah.</p>
     <p class="Clustermoyen">*</p>
-    <p class="Chapitre"><a name="at19_1_25" id="at19_1_25"></a>PSAUME 25*</p>
-    <p class="Sous-Titre">De David.</p>
+    <p class="Chapitre"><a name="at19_1_25" id="at19_1_25"></a>PSAUME 25</p>
+    <p class="Sous-Titre">* De David.</p>
     <p class="Note">— *&#160;: <i>Dans ce psaume et le Ps. 34, la lettre hébraïque initiale de chaque verset suit
     l'ordre alphabétique</i>.</p>
     <p class="Usuel">1 À toi Éternel, j'élève mon âme.</p>
@@ -32864,8 +32866,8 @@
     <p class="Usuel">20 Notre âme s'attend à l'Éternel&#160;; il est notre aide et notre bouclier.</p>
     <p class="Usuel">21 Car notre cœur se réjouira en lui, puisqu'en son saint nom nous avons mis notre confiance.</p>
     <p class="Usuel">22 Que ta bonté, ô Éternel&#160;! soit sur nous, selon que nous nous sommes attendus à toi.</p>
-    <p class="Chapitre"><a name="at19_1_34" id="at19_1_34"></a>PSAUME 34*</p>
-    <p class="Sous-Titre">De David, quand il dissimula sa raison devant Abimélec**, qui le chassa, et il s'en alla.</p>
+    <p class="Chapitre"><a name="at19_1_34" id="at19_1_34"></a>PSAUME 34</p>
+    <p class="Sous-Titre">* De David, quand il dissimula sa raison devant Abimélec**, qui le chassa, et il s'en alla.</p>
     <p class="Note">— *&#160;: <i>voir la note au</i> Psaume 25. — **&#160;: <i>titre des rois philistins</i> (?).</p>
     <p class="Usuel">1 Je bénirai l'Éternel en tout temps&#160;; sa louange sera continuellement dans ma bouche.</p>
     <p class="Usuel">2 Mon âme se glorifiera en l'Éternel&#160;; les débonnaires l'entendront, et se réjouiront.</p>
@@ -32982,8 +32984,8 @@
     <p class="Usuel">12 Là sont tombés les ouvriers d'iniquité&#160;; ils ont été renversés, et n'ont pu se
     relever.</p>
     <p class="Note">— v. 6&#160;: <i>litt.</i>: montagnes de *Dieu.</p>
-    <p class="Chapitre"><a name="at19_1_37" id="at19_1_37"></a>PSAUME 37*</p>
-    <p class="Sous-Titre">De David.</p>
+    <p class="Chapitre"><a name="at19_1_37" id="at19_1_37"></a>PSAUME 37</p>
+    <p class="Sous-Titre">* De David.</p>
     <p class="Note">— *&#160;: <i>La lettre hébraïque initiale de chaque alinéa de ce psaume suit l'ordre
     alphabétique</i>.</p>
     <p class="Usuel">1 Ne t'irrite pas à cause de ceux qui font le mal, ne sois pas jaloux de ceux qui pratiquent
@@ -33206,6 +33208,7 @@
     <p class="AccesDirect">Accès direct aux cinq livres des Psaumes&#160;: <a href="#at19_1">Livre 1</a> (Ps. 1 à 41),
     <a href="#at19_2">Livre 2</a> (Ps. 42 à 72), <a href="#at19_3">Livre 3</a> (Ps. 73 à 89), <a href="#at19_4">Livre
     4</a> (Ps. 90 à 106), <a href="#at19_5">Livre 5</a> (Ps. 107 à 150)</p>
+    <!--
     <p class="Usuel" align="center"><a href="#at19_2_42">42</a>, <a href="#at19_2_43">43</a>, <a href=
     "#at19_2_44">44</a>, <a href="#at19_2_45">45</a>, <a href="#at19_2_46">46</a>, <a href="#at19_2_47">47</a>,
     <a href="#at19_2_48">48</a>, <a href="#at19_2_49">49</a>, <a href="#at19_2_50">50</a>, <a href="#at19_2_51">51</a>,
@@ -33215,6 +33218,7 @@
     <a href="#at19_2_64">64</a>, <a href="#at19_2_65">65</a>, <a href="#at19_2_66">66</a>, <a href="#at19_2_67">67</a>,
     <a href="#at19_2_68">68</a>, <a href="#at19_2_69">69</a>, <a href="#at19_2_70">70</a>, <a href="#at19_2_71">71</a>,
     <a href="#at19_2_72">72</a></p>
+    -->
     <p class="Chapitre"><a name="at19_2_42" id="at19_2_42"></a>PSAUME 42</p>
     <p class="Sous-Titre">Au chef de musique. Instruction. Des fils de Coré.</p>
     <p class="Usuel">1 Comme le cerf brame après les courants d'eau, ainsi mon âme crie* après toi, ô Dieu&#160;!</p>
@@ -34186,11 +34190,13 @@
     <p class="AccesDirect">Accès direct aux cinq livres des Psaumes&#160;: <a href="#at19_1">Livre 1</a> (Ps. 1 à 41),
     <a href="#at19_2">Livre 2</a> (Ps. 42 à 72), <a href="#at19_3">Livre 3</a> (Ps. 73 à 89), <a href="#at19_4">Livre
     4</a> (Ps. 90 à 106), <a href="#at19_5">Livre 5</a> (Ps. 107 à 150)</p>
+    <!--
     <p class="Usuel" align="center"><a href="#at19_3_73">73</a>, <a href="#at19_3_74">74</a>, <a href=
     "#at19_3_75">75</a>, <a href="#at19_3_76">76</a>, <a href="#at19_3_77">77</a>, <a href="#at19_3_78">78</a>,
     <a href="#at19_3_79">79</a>, <a href="#at19_3_80">80</a>, <a href="#at19_3_81">81</a>, <a href="#at19_3_82">82</a>,
     <a href="#at19_3_83">83</a>, <a href="#at19_3_84">84</a>, <a href="#at19_3_85">85</a>, <a href="#at19_3_86">86</a>,
     <a href="#at19_3_87">87</a>, <a href="#at19_3_88">88</a>, <a href="#at19_3_89">89</a></p>
+    -->
     <p class="Chapitre"><a name="at19_3_73" id="at19_3_73"></a>PSAUME 73</p>
     <p class="Sous-Titre">Psaume d'Asaph.</p>
     <p class="Usuel">1 Certainement Dieu est bon envers Israël, envers ceux qui sont purs de cœur.</p>
@@ -34864,12 +34870,14 @@
     <p class="AccesDirect">Accès direct aux cinq livres des Psaumes&#160;: <a href="#at19_1">Livre 1</a> (Ps. 1 à 41),
     <a href="#at19_2">Livre 2</a> (Ps. 42 à 72), <a href="#at19_3">Livre 3</a> (Ps. 73 à 89), <a href="#at19_4">Livre
     4</a> (Ps. 90 à 106), <a href="#at19_5">Livre 5</a> (Ps. 107 à 150)</p>
+    <!--
     <p class="Usuel" align="center"><a href="#at19_4_090">90</a>, <a href="#at19_4_091">91</a>, <a href=
     "#at19_4_092">92</a>, <a href="#at19_4_093">93</a>, <a href="#at19_4_094">94</a>, <a href="#at19_4_095">95</a>,
     <a href="#at19_4_096">96</a>, <a href="#at19_4_097">97</a>, <a href="#at19_4_098">98</a>, <a href=
     "#at19_4_099">99</a>, <a href="#at19_4_100">100</a>, <a href="#at19_4_101">101</a>, <a href="#at19_4_102">102</a>,
     <a href="#at19_4_103">103</a>, <a href="#at19_4_104">104</a>, <a href="#at19_4_105">105</a>, <a href=
     "#at19_4_106">106</a></p>
+    -->
     <p class="Chapitre"><a name="at19_4_090" id="at19_4_090"></a>PSAUME 90</p>
     <p class="Sous-Titre">Prière de Moïse, homme de Dieu.</p>
     <p class="Usuel">1 Seigneur, tu as été notre demeure de génération en génération.</p>
@@ -35454,6 +35462,7 @@
     <p class="AccesDirect">Accès direct aux cinq livres des Psaumes&#160;: <a href="#at19_1">Livre 1</a> (Ps. 1 à 41),
     <a href="#at19_2">Livre 2</a> (Ps. 42 à 72), <a href="#at19_3">Livre 3</a> (Ps. 73 à 89), <a href="#at19_4">Livre
     4</a> (Ps. 90 à 106), <a href="#at19_5">Livre 5</a> (Ps. 107 à 150)</p>
+    <!--
     <p class="Usuel" align="center"><a href="#at19_5_107">107</a>, <a href="#at19_5_108">108</a>, <a href=
     "#at19_5_109">109</a>, <a href="#at19_5_110">110</a>, <a href="#at19_5_111">111</a>, <a href="#at19_5_112">112</a>,
     <a href="#at19_5_113">113</a>, <a href="#at19_5_114">114</a>, <a href="#at19_5_115">115</a>, <a href=
@@ -35467,6 +35476,7 @@
     <a href="#at19_5_141">141</a>, <a href="#at19_5_142">142</a>, <a href="#at19_5_143">143</a>, <a href=
     "#at19_5_144">144</a>, <a href="#at19_5_145">145</a>, <a href="#at19_5_146">146</a>, <a href="#at19_5_147">147</a>,
     <a href="#at19_5_148">148</a>, <a href="#at19_5_149">149</a>, <a href="#at19_5_150">150</a></p>
+    -->
     <p class="Chapitre"><a name="at19_5_107" id="at19_5_107"></a>PSAUME 107</p>
     <p class="Usuel">1 Célébrez l'Éternel&#160;! Car il est bon&#160;; car sa bonté demeure à toujours*.</p>
     <p class="Usuel">2 Que les rachetés de l'Éternel le disent, ceux qu'il a rachetés de la main de l'oppresseur,</p>
@@ -35644,10 +35654,10 @@
     <p class="Usuel">7 Il boira du torrent dans le chemin, c'est pourquoi il lèvera haut la tête.</p>
     <p class="Note">— v. 6&#160;: <i>litt.</i>: sur un.</p>
     <p class="Clustermoyen">*</p>
-    <p class="Chapitre"><a name="at19_5_111" id="at19_5_111"></a>PSAUME 111*</p>
-    <p class="Sous-Titre">Louez Jah. **</p>
+    <p class="Chapitre"><a name="at19_5_111" id="at19_5_111"></a>PSAUME 111</p>
+    <p class="Sous-Titre">* Louez Jah. **</p>
     <p class="Note">— <i>*</i>&#160;: <i>La lettre hébraïque initiale de chacun des moitiés des versets des</i> Psaumes
-    111 <i>et</i> 112 <i>suit l'ordre alphabétique. —</i> **&#160;: <i>autrement dit</i>&#160;: Alléluia<i>.</i></p>
+    111 <i>et</i> 112 <i>suit l'ordre alphabétique.</i> — **&#160;: <i>autrement dit</i>&#160;: Alléluia<i>.</i></p>
     <p class="Usuel">1 Je célébrerai l'Éternel de tout mon cœur, dans la compagnie* des hommes droits et dans
     l'assemblée.</p>
     <p class="Note">— v. 1&#160;: <i>ailleurs</i>&#160;: conseil secret.</p>
@@ -35667,10 +35677,10 @@
     est saint et terrible.</p>
     <p class="Usuel">10 La crainte de l'Éternel est le commencement de la sagesse&#160;; tous ceux qui pratiquent [ses
     préceptes] auront une bonne intelligence. Sa louange demeure à perpétuité.</p>
-    <p class="Chapitre"><a name="at19_5_112" id="at19_5_112"></a>PSAUME 112*</p>
-    <p class="Sous-Titre">Louez Jah. **</p>
+    <p class="Chapitre"><a name="at19_5_112" id="at19_5_112"></a>PSAUME 112</p>
+    <p class="Sous-Titre">* Louez Jah. **</p>
     <p class="Note">— <i>*</i>&#160;: <i>La lettre hébraïque initiale de chacun des moitiés des versets des</i> Psaumes
-    111 <i>et</i> 112 <i>suit l'ordre alphabétique. —</i> **&#160;: <i>autrement dit</i>&#160;: Alléluia<i>.</i></p>
+    111 <i>et</i> 112 <i>suit l'ordre alphabétique.</i> — **&#160;: <i>autrement dit</i>&#160;: Alléluia<i>.</i></p>
     <p class="Usuel">1 Bienheureux l'homme qui craint l'Éternel [et] qui prend un grand plaisir en ses
     commandements&#160;!</p>
     <p class="Usuel">2 Sa semence sera puissante dans le pays&#160;; … la génération des hommes droits sera bénie.</p>
@@ -35832,7 +35842,8 @@
     <p class="Usuel">29 Célébrez l'Éternel&#160;! car il est bon, car sa bonté demeure à toujours.</p>
     <p class="Note">— v. 27&#160;: <i>litt.</i>: [sacrifice de] fête.</p>
     <p class="Clustermoyen">*</p>
-    <p class="Chapitre"><a name="at19_5_119" id="at19_5_119"></a>PSAUME 119*</p>
+    <p class="Chapitre"><a name="at19_5_119" id="at19_5_119"></a>PSAUME 119</p>
+    <p class="Sous-Titre">*</p>
     <p class="Note">— *&#160;: <i>Ce Psaume est divisé en paragraphes de huit versets chacun, la lettre hébraïque
     initiale de chaque verset dans le paragraphe restant la même, et suivant, pour ceux-ci, l'ordre
     alphabétique</i>.</p>
@@ -36562,8 +36573,8 @@
     <p class="Usuel">15 Bienheureux le peuple pour qui il en est ainsi&#160;! Bienheureux le peuple qui a l'Éternel
     pour son Dieu&#160;!</p>
     <p class="Note">— v. 14&#160;: <i>ou</i>&#160;: nos bœufs soient chargés.</p>
-    <p class="Chapitre"><a name="at19_5_145" id="at19_5_145"></a>PSAUME 145*</p>
-    <p class="Sous-Titre">Louange de David.</p>
+    <p class="Chapitre"><a name="at19_5_145" id="at19_5_145"></a>PSAUME 145</p>
+    <p class="Sous-Titre">* Louange de David.</p>
     <p class="Note">— *&#160;: <i>La lettre hébraïque initiale de chaque verset de ce psaume suit l'ordre
     alphabétique</i>.</p>
     <p class="Usuel">1 Je t'exalterai, mon Dieu, ô Roi&#160;! et je bénirai ton nom à toujours et à perpétuité.</p>
@@ -39284,7 +39295,8 @@
     <p class="Usuel">14 car Dieu amènera toute œuvre en jugement, avec tout ce qui est caché, soit bien, soit mal.</p>
     <p class="Usuel">&#160;</p>
     <p class="Usuel">&#160;</p>
-    <p class="Livre"><a name="at22" id="at22"></a>LE CANTIQUE DES CANTIQUES*</p>
+    <p class="Livre"><a name="at22" id="at22"></a>LE CANTIQUE DES CANTIQUES</p>
+    <p class="Sous-Titre">*</p>
     <p class="Note">— *&#160;: <i>En général, dans ce livre, les alinéas marquent la succession des
     interlocuteurs.</i></p>
     <p class="Usuel">&#160;</p>
@@ -40673,7 +40685,7 @@
     <p class="Usuel">16 Du bout du pays* nous avons entendu des chants&#160;: Gloire au juste&#160;!</p>
     <p class="Note">— v. 14&#160;: les épargnés de la dispersion d'Israël. — v. 15*&#160;: <i>mot très-différemment
     interprété</i>. — v. 15**&#160;: <i>ou</i>&#160;: de la mer. — v. 16&#160;: <i>ou</i>&#160;: de la terre.</p>
-    <p class="Usuel">Et j'ai dit&#160;: Ma maigreur*, ma maigreur*, malheur à moi&#160;! Les perfides ont agi
+    <p class="Usuel">Et j'ai dit&#160;: Ma maigreur**, ma maigreur***, malheur à moi&#160;! Les perfides ont agi
     perfidement, les perfides ont agi avec une insigne perfidie.</p>
     <p class="Usuel">17 La frayeur, et la fosse, et le piège, sont sur toi, habitant du pays.</p>
     <p class="Usuel">18 Et il arrivera que celui qui s'enfuit de devant le cri de la frayeur tombera dans la fosse, et
@@ -40682,7 +40694,7 @@
     <p class="Usuel">19 La terre est entièrement brisée, la terre se dissout, la terre est violemment remuée&#160;;</p>
     <p class="Usuel">20 la terre chancelle, elle chancelle comme un homme ivre&#160;; elle est ébranlée deçà et delà
     comme une cabane pour la nuit&#160;; sa transgression pèse sur elle&#160;: elle tombera et ne se relèvera pas.</p>
-    <p class="Note">— v. 16&#160;: <i>ou</i>&#160;: Maigreur à moi&#160;; <i>selon quelques-uns</i>&#160;: mon
+    <p class="Note">— v. 16**, 16***&#160;: <i>ou</i>&#160;: Maigreur à moi&#160;; <i>selon quelques-uns</i>&#160;: mon
     secret.</p>
     <p class="Usuel">21 Et il arrivera, en ce jour-là, que l'Éternel visitera* l'armée d'en haut, en haut, et les rois
     de la terre, sur la terre.</p>
@@ -41748,7 +41760,7 @@
     et ne dit pas&#160;: N'ai-je pas un mensonge dans ma main droite&#160;?</p>
     <p class="Note">— v. 9*&#160;: <i>ailleurs</i>&#160;: le vide. — v. 9**&#160;: <i>selon quelques-uns</i>&#160;:
     oui, elles sont des témoins pour eux&#160;: elles ne voient pas et elles…. — v. 12*&#160;: <i>d'autres</i>&#160;:
-    une hache. — v. 12**&#160;: <i>litt.</i>: la. — v. 13 <i>ou</i>&#160;: avec un poinçon. — v. 14*&#160;:
+    une hache. — v. 12**&#160;: <i>litt.</i>: la. — v. 13&#160;: <i>ou</i>&#160;: avec un poinçon. — v. 14*&#160;:
     <i>litt.</i>: choisit (<i>ou</i> fortifie) pour lui. — v. 14**&#160;: <i>ou</i>&#160;: laurier (?). — v. 16&#160;:
     <i>litt.</i>: la lumière.</p>
     <p class="Usuel">21 Souviens-toi de ces choses, ô Jacob, et toi, Israël, car tu es mon serviteur. Je t'ai
@@ -45963,7 +45975,7 @@
     <p class="Usuel">10 L'Éternel a produit au jour* notre justice**. Venez et racontons en Sion l'œuvre de l'Éternel,
     notre Dieu.</p>
     <p class="Note">— v. 10*&#160;: <i>litt</i>.: a fait sortir. — v. 10**&#160;: <i>hébreu</i>&#160;: nos
-    justices&#160;; <i>comparer</i> És. 64:6.</p>
+    justices&#160;; <i>comparer</i> Ésaïe 64:6.</p>
     <p class="Usuel">11 Aiguisez les flèches, prenez* les boucliers&#160;! L'Éternel a réveillé l'esprit des rois des
     Mèdes, car sa pensée est contre Babylone pour la détruire, car c'est ici la vengeance de l'Éternel, la vengeance de
     son temple.</p>
@@ -46037,7 +46049,7 @@
     <p class="Usuel">39 Quand ils seront échauffés, je préparerai leur breuvage, et je les enivrerai, afin qu'ils
     s'égayent, et qu'ils dorment d'un sommeil perpétuel et ne se réveillent plus, dit l'Éternel.</p>
     <p class="Usuel">40 Je les ferai descendre comme des agneaux à la tuerie, comme des béliers avec des boucs.</p>
-    <p class="Note">— v. 36*&#160;: <i>comparer</i> És. 34:8. — v. 36**&#160;: <i>comparer la note à</i> És. 19:5. — v.
+    <p class="Note">— v. 36*&#160;: <i>comparer</i> Ésaïe 34:8. — v. 36**&#160;: <i>comparer la note à</i> Ésaïe 19:5. — v.
     37&#160;: <i>ou</i>&#160;: une désolation.</p>
     <p class="Usuel">41 Comment a été prise Shéshac*, et comment la louange de toute la terre a-t-elle été
     saisie&#160;! Comment Babylone est-elle devenue un sujet d'étonnement** parmi les nations&#160;!</p>
@@ -46189,11 +46201,11 @@
     <p class="Usuel">&#160;</p>
     <p class="AccesDirect">Accès direct aux chapitres des Lamentations de Jérémie&#160;: <a href="#at25_01">1</a>,
     <a href="#at25_02">2</a>, <a href="#at25_03">3</a>, <a href="#at25_04">4</a>, <a href="#at25_05">5</a></p>
-    <p class="Usuel">&#160;</p>
-    <p class="Note">— <i>La lettre hébraïque initiale de chacun des versets des chapitres 1, 2 et 4, suit l'ordre
+    <p class="Chapitre"><a name="at25_01" id="at25_01"></a>Chapitre 1</p>
+    <p class="Sous-Titre">*</p>
+    <p class="Note">— *&#160;: <i>La lettre hébraïque initiale de chacun des versets des chapitres 1, 2 et 4, suit l'ordre
     alphabétique&#160;; au chapitre 3, il en est de même pour chaque paragraphe de trois versets, la lettre initiale de
     chacun de ceux-ci restant la même.</i></p>
-    <p class="Chapitre"><a name="at25_01" id="at25_01"></a>Chapitre 1</p>
     <p class="Posie">1 Comment est-elle assise solitaire, la ville si peuplée&#160;! Celle qui était grande entre les
     nations est devenue comme veuve&#160;; la princesse parmi les provinces est devenue tributaire.</p>
     <p class="Posie">2 Elle pleure, elle pleure pendant la nuit, et ses larmes sont sur ses joues&#160;; de tous ses
@@ -46937,7 +46949,7 @@
     <p class="Usuel">9 Et il me dit&#160;: L'iniquité de la maison d'Israël et de Juda est excessivement grande, et le
     pays est rempli de sang, et la ville est remplie d'injustices&#160;; car ils ont dit&#160;: L'Éternel a abandonné
     le pays*, et l'Éternel ne voit pas.</p>
-    <p class="Usuel">10&#160;Et moi aussi, — mon œil n'aura point compassion, et je n'épargnerai pas&#160;; je ferai
+    <p class="Usuel">10 Et moi aussi, — mon œil n'aura point compassion, et je n'épargnerai pas&#160;; je ferai
     retomber* leur voie sur leur tête.</p>
     <p class="Usuel">11 Et voici, l'homme vêtu de lin, qui avait l'encrier à ses reins, rapporta, disant&#160;: J'ai
     fait comme tu m'as commandé.</p>
@@ -51295,7 +51307,7 @@
     trembleront&#160;; et l'Éternel sera l'abri de son peuple et le refuge des fils d'Israël.</p>
     <p class="Usuel">17 Et vous saurez que moi, l'Éternel, je suis votre Dieu, qui demeure en Sion, ma montagne sainte.
     Et Jérusalem sera sainte, et les étrangers n'y passeront plus.</p>
-    <p class="Note">— v. 14&#160;: <i>plutôt</i>&#160;: ce qui est décrété, <i>comparer</i> És. 10:22.</p>
+    <p class="Note">— v. 14&#160;: <i>plutôt</i>&#160;: ce qui est décrété, <i>comparer</i> Ésaïe 10:22.</p>
     <p class="Usuel">18 Et il arrivera, en ce jour-là, que les montagnes ruisselleront de moût, et les collines
     découleront de lait, et tous les torrents de Juda découleront d'eau&#160;; et une source sortira de la maison de
     l'Éternel, et arrosera la vallée de Sittim*.</p>
@@ -51686,7 +51698,7 @@
     <p class="Usuel">&#160;</p>
     <p class="Usuel">&#160;</p>
     <p class="Livre"><a name="at31" id="at31"></a>ABDIAS</p>
-    <p class="Usuel">&#160;</p>
+    <p class="Chapitre"><a name="at31_01" id="at31_01"/>Chapitre 1</p>
     <p class="Usuel">1 La vision d'Abdias.* — Ainsi dit le Seigneur, l'Éternel, touchant Édom. Nous avons entendu une
     rumeur de par l'Éternel, et un ambassadeur a été envoyé parmi les nations&#160;: Levez-vous&#160;! et levons-nous
     contre lui pour la guerre.</p>
@@ -51808,7 +51820,7 @@
     <p class="Posie">10 Mais moi, je te sacrifierai avec une voix de louange&#160;; je m'acquitterai de ce que j'ai
     voué. La délivrance est de l'Éternel.</p>
     <p class="Usuel">11 Et l'Éternel commanda au poisson, et il vomit Jonas sur la terre*.</p>
-    <p class="Note">— v. 8&#160;: <i>ou</i>&#160;: palais. — v. 9&#160;: <i>au Psaume</i> 31:6&#160;: vaines idoles. —
+    <p class="Note">— v. 8&#160;: <i>ou</i>&#160;: palais. — v. 9&#160;: <i>au Psaume 31:6</i>&#160;: vaines idoles. —
     v. 11&#160;: <i>litt</i>.: le sec.</p>
     <p class="Chapitre"><a name="at32_03" id="at32_03"></a>Chapitre 3</p>
     <p class="Usuel">1 Et la parole de l'Éternel [vint] à Jonas une seconde fois, disant&#160;:</p>
@@ -52794,12 +52806,12 @@
     vingt coudées, et large de dix coudées.</p>
     <p class="Usuel">3 Et il me dit&#160;: C'est ici la malédiction* qui sort sur la face de toute la terre**&#160;;
     car tout voleur sera détruit, selon ce [qui est écrit]***, d'une part&#160;; et chacun qui jure sera détruit, selon
-    ce [qui est écrit]***, de l'autre part.</p>
+    ce [qui est écrit]****, de l'autre part.</p>
     <p class="Usuel">4 Je la fais sortir, dit l'Éternel des armées, et elle entrera dans la maison du voleur, et dans
     la maison de celui qui jure faussement par mon nom&#160;; et elle logera au milieu de sa maison et la consumera
     avec le bois et les pierres.</p>
     <p class="Note">— v. 3*&#160;: <i>ailleurs</i>&#160;: exécration. — v. 3**&#160;: <i>litt</i>.: tout le pays. — v.
-    3***&#160;: <i>litt</i>.: selon elle.</p>
+    3***, 3****&#160;: <i>litt</i>.: selon elle.</p>
     <p class="Usuel">5 Et l'ange qui parlait avec moi sortit et me dit&#160;: Lève tes yeux et regarde&#160;: Qu'est-ce
     qui sort là&#160;?</p>
     <p class="Usuel">6 Et je dis&#160;: Qu'est-ce&#160;? Et il dit&#160;: C'est l'épha qui sort. Et il dit&#160;: C'est
@@ -53087,7 +53099,7 @@
     <p class="Usuel">13 la famille de la maison de Lévi à part, et leurs femmes à part&#160;; la famille des Shimhites*
     à part, et leurs femmes à part&#160;:</p>
     <p class="Usuel">14 toutes les familles qui seront de reste, chaque famille à part, et leurs femmes à part.</p>
-    <p class="Note">— v. 11&#160;: <i>comparer la note à</i> Josué 11:8. — v. 13 <i>voir</i> Nombres 3:21 (?).</p>
+    <p class="Note">— v. 11&#160;: <i>comparer la note à</i> Josué 11:8. — v. 13&#160;: <i>voir</i> Nombres 3:21 (?).</p>
     <p class="Chapitre"><a name="at38_13" id="at38_13"></a>Chapitre 13</p>
     <p class="Usuel">1 En ce jour-là, une source sera ouverte pour la maison de David et pour les habitants de
     Jérusalem, pour le péché et pour l'impureté.</p>
@@ -53585,7 +53597,7 @@
     hériter de la terre, <i>et vice versa</i>.</p>
     <p class="Usuel">13 Vous êtes le sel de la terre*&#160;; mais si le sel a perdu sa saveur, avec quoi sera-t-il
     salé&#160;? Il n'est plus bon à rien qu'à être jeté dehors et à être foulé aux pieds par les hommes.</p>
-    <p class="Note">— v. 13&#160;: <i>ou</i>&#160;: pays.</p>
+    <p class="Usuel">&#160;</p>
     <p class="Usuel">14 Vous êtes la lumière du monde&#160;: une ville située sur une montagne ne peut être cachée.</p>
     <p class="Usuel">15 Aussi n'allume-t-on pas une lampe pour la mettre ensuite sous le boisseau, mais sur le pied de
     lampe&#160;; et elle luit pour tous ceux qui sont dans la maison.</p>
@@ -56240,7 +56252,7 @@
     <p class="Usuel"><a name="nt02_09" id="nt02_09"></a><b>Chapitre 9.</b> 1 Et il leur dit&#160;: En vérité, je vous
     dis, que de ceux qui sont ici présents, il y en a quelques-uns qui ne goûteront point la mort jusqu'à ce qu'ils
     aient vu le royaume de Dieu venu avec puissance.</p>
-    <p class="Note">— v. 35 <i>à</i> 37&#160;: <i>le terme original signifie à la fois</i>&#160;: vie <i>et</i>
+    <p class="Note">— v. 35, 36, 37&#160;: <i>le terme original signifie à la fois</i>&#160;: vie <i>et</i>
     âme.</p>
     <p class="Usuel">2 Et après six jours, Jésus prend avec lui Pierre et Jacques et Jean, et les mène seuls à l'écart,
     sur une haute montagne. Et il fut transfiguré devant eux&#160;;</p>
@@ -57029,7 +57041,7 @@
     une vision dans le temple*&#160;; et lui-même leur faisait des signes, et il demeura muet.</p>
     <p class="Usuel">23 Et il arriva que, quand les jours de son ministère furent accomplis, il s'en alla dans sa
     maison.</p>
-    <p class="Note">— v. 9 et 21 et 22&#160;: la maison même&#160;; <i>voir</i> Matthieu 23:16. — v. 13&#160;: <i>en
+    <p class="Note">— v. 9, 21, 22&#160;: la maison même&#160;; <i>voir</i> Matthieu 23:16. — v. 13&#160;: <i>en
     hébreu</i>&#160;: l'Éternel a fait grâce. — v. 15&#160;: <i>plutôt</i>&#160;: ni aucune boisson fermentée. — v.
     17&#160;: <i>ou</i>&#160;: selon. — v. 19&#160;: <i>litt</i>.: t'évangéliser ces choses.</p>
     <p class="Usuel">24 Or après ces jours, Élisabeth sa femme conçut, et elle se cacha cinq mois, disant&#160;:</p>
@@ -57988,7 +58000,7 @@
     j'aie fait pour hériter de la vie éternelle&#160;?</p>
     <p class="Usuel">26 Et il lui dit&#160;: Qu'est-il écrit dans la loi&#160;? Comment lis-tu&#160;?</p>
     <p class="Usuel">27 Et répondant, il dit&#160;: «Tu aimeras le *Seigneur ton Dieu de tout ton cœur, et de toute ton
-    âme, et de toute ta force, et de toute ta pensée» <i>[Deutéronome 6:5]&#160;</i>; «et ton prochain comme toi-même»
+    âme, et de toute ta force, et de toute ta pensée» <i>[Deutéronome 6:5]</i>&#160;; «et ton prochain comme toi-même»
     <i>[Lévitique 19:18]</i>.</p>
     <p class="Usuel">28 Et il lui dit&#160;: Tu as bien répondu&#160;; fais cela, et tu vivras.</p>
     <p class="Usuel">29 Mais lui, voulant se justifier lui-même, dit à Jésus&#160;: Et qui est mon prochain&#160;?</p>
@@ -58236,7 +58248,7 @@
     contre la mère&#160;; la belle-mère contre sa belle-fille, et la belle-fille contre sa belle-mère.</p>
     <p class="Note">— v. 24&#160;: <i>ou</i>&#160;: car ils. — v. 25 et 52&#160;: une seule. — v. 32&#160;:
     <i>proprement</i>&#160;: [toi] le petit troupeau. — v. 35, 37**&#160;: <i>les vêtements orientaux, très-larges,
-    devaient être retenus par une ceinture pour marcher ou agir</i>. — v. 36, 37*, 42 et suivants&#160;:
+    devaient être retenus par une ceinture pour marcher ou agir</i>. — v. 36, 37*, 42&#160;:
     ordinairement&#160;: seigneur. — v. 38&#160;: <i>la nuit était divisée en 4 périodes, ou veilles&#160;; voir</i>
     Marc 13:35. — v. 48&#160;: <i>litt</i>.: de lui il sera. — v. 49&#160;: <i>ou</i>&#160;: que puis-je désirer.</p>
     <p class="Usuel">54 Et il dit aussi aux foules&#160;: Quand vous voyez une nuée se lever de l'occident, aussitôt
@@ -59304,7 +59316,7 @@
     <p class="Usuel">4 En elle était [la] vie, et la vie était la lumière des hommes*.</p>
     <p class="Usuel">5 Et la lumière luit dans les ténèbres&#160;; et les ténèbres ne l'ont pas comprise.</p>
     <p class="Note">— v. 1&#160;: <i>ou</i>&#160;: le Verbe (<i>mot masculin dans l'original</i>). — v. 2&#160;:
-    <i>ou</i>&#160;: Il (<i>le Verbe</i>). — v. 4 <i>ou aussi</i>&#160;: la lumière des hommes était la vie.</p>
+    <i>ou</i>&#160;: Il (<i>le Verbe</i>). — v. 4&#160;: <i>ou aussi</i>&#160;: la lumière des hommes était la vie.</p>
     <p class="Usuel">6 Il y eut un homme envoyé de Dieu&#160;; son nom était Jean.</p>
     <p class="Usuel">7 Celui-ci vint pour [rendre] témoignage, pour rendre témoignage de* la lumière, afin que tous
     crussent par lui.</p>
@@ -59378,7 +59390,7 @@
     tu seras appelé Céphas (qui est interprété Pierre*).</p>
     <p class="Note">— v. 39&#160;: <i>ici, et</i> <i>ailleurs</i> <i>souvent</i>&#160;: celui qui enseigne. — v.
     40&#160;: <i>c'est-à-dire &#160;vers la fin de la journée (divisée en douze heures)</i>. — v. 42&#160;:
-    <i>ou</i>&#160;: Oint&#160;; <i>voir</i> Psaume 2:2, 6&#160;; Matthieu 1:16. — v. 43 <i>ou</i>&#160;: pierre.</p>
+    <i>ou</i>&#160;: Oint&#160;; <i>voir</i> Psaume 2:2, 6&#160;; Matthieu 1:16. — v. 43&#160;: <i>ou</i>&#160;: pierre.</p>
     <p class="Usuel">44 Le lendemain, il voulut s'en aller en Galilée. Et Jésus trouve Philippe, et lui dit&#160;:
     Suis-moi.</p>
     <p class="Usuel">45 Or Philippe était de Bethsaïda, de la ville d'André et de Pierre.</p>
@@ -59992,7 +60004,7 @@
     <p class="Usuel">29 Et celui qui m'a envoyé est avec moi&#160;; il ne m'a pas laissé seul, parce que moi, je fais
     toujours les choses qui lui plaisent.</p>
     <p class="Usuel">30 Comme il disait ces choses, plusieurs crurent en lui.</p>
-    <p class="Note">— v. 23*&#160;: <i>litt</i>.: de ces choses qui sont en bas. — v. 23** <i>litt</i>.: de ces choses
+    <p class="Note">— v. 23*&#160;: <i>litt</i>.: de ces choses qui sont en bas. — v. 23**&#160;: <i>litt</i>.: de ces choses
     qui sont en haut. — v. 25&#160;: dans le principe et l'universalité de ce que je suis. Sa parole, son langage, le
     présentait lui-même, — étant la vérité. — v. 28&#160;: <i>c'est-à-dire</i> &#160;: qui je suis (le Fils et l'Envoyé
     du Père).</p>
@@ -60055,7 +60067,7 @@
     <p class="Note">— v. 38&#160;: faire, <i>ici</i>, faire habituellement <i>ou</i> de manière caractéristique&#160;;
     <i>ailleurs</i>&#160;: pratiquer. — v. 40&#160;: <i>litt</i>.: parlé. — v. 41&#160;: un seul. — v. 44&#160;:
     <i>litt</i>.: ne s'est pas tenu debout. — v. 46&#160;: <i>c'est-à-dire</i> &#160;de défaillance dans
-    l'accomplissement de ma mission. — v. 55* connaître, objectivement, — pour les Juifs — v. 55** connaître&#160;:
+    l'accomplissement de ma mission. — v. 55*&#160;: connaître, objectivement, — pour les Juifs — v. 55**&#160;: connaître&#160;:
     <i>ici, et jusqu'à la fin du verset</i>&#160;: connaître subjectivement. — v. 56&#160;: <i>ou</i>&#160;: de voir. —
     v. 58&#160;: vint à l'existence.</p>
     <p class="Chapitre"><a name="nt04_09" id="nt04_09"></a>Chapitre 9</p>
@@ -60366,7 +60378,7 @@
     conservera pour la vie éternelle.</p>
     <p class="Usuel">26 Si quelqu'un me sert, qu'il me suive&#160;; et où je suis, moi, là aussi sera mon
     serviteur&#160;: si quelqu'un me sert, le Père l'honorera.</p>
-    <p class="Note">— v. 20 <i>ordinairement</i>&#160;: rendre hommage&#160;; <i>se dit à l'égard de Dieu et à l'égard
+    <p class="Note">— v. 20&#160;: <i>ordinairement</i>&#160;: rendre hommage&#160;; <i>se dit à l'égard de Dieu et à l'égard
     des hommes</i>. — v. 21&#160;: <i>plutôt</i>&#160;: Monsieur.</p>
     <p class="Usuel">27 Maintenant mon âme est troublée&#160;; et que dirai-je&#160;? Père, délivre-moi de cette
     heure&#160;; mais c'est pour cela que je suis venu à* cette heure.</p>
@@ -60387,7 +60399,7 @@
     marche dans les ténèbres ne sait où il va.</p>
     <p class="Usuel">36 Pendant que vous avez la lumière, croyez en la lumière, afin que vous soyez fils de lumière.
     Jésus dit ces choses, et s'en allant, il se cacha de devant eux.</p>
-    <p class="Note">— v. 27 <i>c'est-à-dire</i> &#160;jusqu'à. — v. 31&#160;: Satan&#160;: <i>comparer</i> 14:30&#160;;
+    <p class="Note">— v. 27&#160;: <i>c'est-à-dire</i> &#160;jusqu'à. — v. 31&#160;: Satan&#160;: <i>comparer</i> 14:30&#160;;
     16:11. — v. 33&#160;: <i>voir</i> 18:32. — v. 34&#160;: <i>proprement</i>&#160;: entendu dire.</p>
     <p class="Usuel">37 Et quoiqu'il eût fait tant de miracles* devant eux, ils ne crurent pas en lui&#160;;</p>
     <p class="Usuel">38 afin que la parole d'Ésaïe le prophète, qu'il prononça, fût accomplie&#160;: «*Seigneur, qui
@@ -61191,7 +61203,7 @@
     Jésus que vous avez crucifié.</p>
     <p class="Note">— v. 15&#160;: <i>le milieu de la matinée</i>. — v. 18&#160;: <i>ailleurs aussi</i>&#160;:
     esclaves. — v. 19&#160;: <i>litt</i>.: je donnerai. — v. 22&#160;: <i>à la fois révélé et consacré, d'où</i>
-    accrédité. — v. 23 <i>ou</i>&#160;: hommes sans loi. — v. 27*, 31&#160;: le hadès est une expression très-vague,
+    accrédité. — v. 23&#160;: <i>ou</i>&#160;: hommes sans loi. — v. 27*, 31&#160;: le hadès est une expression très-vague,
     <i>comme</i> shéol dans l'Ancien Testament, qui désigne le lieu invisible où les âmes des hommes vont après la
     mort&#160;; le hadès est distinct de la géhenne qui est le lieu des tourments infernaux. — v. 27**&#160;:
     <i>ou</i>&#160;: pieux&#160;; <i>voir</i> 13:35 <i>et</i> 2&#160;Chroniques 6:42&#160;; <i>correspond aux mots
@@ -61680,13 +61692,13 @@
     <p class="Usuel">35 Et Philippe, ouvrant sa bouche et commençant par cette écriture, lui annonça Jésus.</p>
     <p class="Usuel">36 Et comme ils continuaient leur chemin, ils arrivèrent à une eau, et l'eunuque dit&#160;: Voici
     de l'eau, qu'est-ce qui m'empêche d'être baptisé&#160;?</p>
-    <p class="Usuel">* 38 Et il donna l'ordre qu'on arrêtât le char, et ils descendirent tous deux à l'eau, et Philippe
+    <p class="Usuel">38 *Et il donna l'ordre qu'on arrêtât le char, et ils descendirent tous deux à l'eau, et Philippe
     et l'eunuque&#160;; et [Philippe] le baptisa.</p>
     <p class="Usuel">39 Et, quand ils furent remontés hors de l'eau, l'Esprit du Seigneur enleva Philippe, et l'eunuque
     ne le vit plus, car il continua son chemin tout joyeux&#160;;</p>
     <p class="Usuel">40 mais Philippe fut trouvé à Azot&#160;; et en passant au travers [du pays], il évangélisa toutes
     les villes, jusqu'à ce qu'il fut arrivé à Césarée.</p>
-    <p class="Note">— v. 37 <i>le «&#160;texte reçu&#160;» ajoute</i>&#160;: Et Philippe dit&#160;: Si tu crois de tout
+    <p class="Note">— v. 38&#160;: <i>le «&#160;texte reçu&#160;» ajoute</i>&#160;: (v.37) Et Philippe dit&#160;: Si tu crois de tout
     ton cœur, [cela] est permis. Et répondant, il dit&#160;: Je crois que Jésus Christ est le Fils de Dieu.</p>
     <p class="Chapitre"><a name="nt05_09" id="nt05_09"></a>Chapitre 9</p>
     <p class="Usuel">1 Or Saul, respirant encore menace et meurtre contre les disciples du Seigneur, alla au souverain
@@ -62208,7 +62220,7 @@
     choses-ci qui sont nécessaires&#160;:</p>
     <p class="Usuel">29 qu'on s'abstienne des choses sacrifiées aux idoles, et du sang, et de ce qui est étouffé, et de
     la fornication. Si vous vous gardez de ces choses, vous ferez* bien. Portez-vous bien.</p>
-    <p class="Note">— v. 25 <i>ou</i>&#160;: étant assemblés d'un commun accord. — v. 29&#160;: <i>ou</i>&#160;:
+    <p class="Note">— v. 25&#160;: <i>ou</i>&#160;: étant assemblés d'un commun accord. — v. 29&#160;: <i>ou</i>&#160;:
     irez.</p>
     <p class="Usuel">30 Eux donc ayant été congédiés, vinrent à Antioche, et ayant assemblé la multitude, ils remirent
     la lettre.</p>
@@ -62217,9 +62229,9 @@
     discours et les fortifièrent.</p>
     <p class="Usuel">33 Et après avoir séjourné là quelque temps, ils furent renvoyés en paix par les frères vers ceux
     qui les avaient envoyés.</p>
-    <p class="Usuel">* 35 Et Paul et Barnabas séjournèrent à Antioche, enseignant et annonçant, avec plusieurs autres
+    <p class="Usuel">35 *Et Paul et Barnabas séjournèrent à Antioche, enseignant et annonçant, avec plusieurs autres
     aussi, la parole du Seigneur.</p>
-    <p class="Note">— v. 34&#160;: <i>ajouté par le texte «reçu»</i>&#160;: Mais il sembla bon à Silas de demeurer
+    <p class="Note">— v. 35&#160;: <i>ajouté par le texte «reçu»</i>&#160;: (v.34) Mais il sembla bon à Silas de demeurer
     là.</p>
     <p class="Usuel">36 Et quelques jours après, Paul dit à Barnabas&#160;: Retournons maintenant visiter les frères
     par toutes les villes où nous avons annoncé la parole du Seigneur, [pour voir] comment ils vont.</p>
@@ -62748,8 +62760,8 @@
     <p class="Usuel">20 et lorsque le sang d'Étienne, ton témoin, fut répandu, moi-même aussi j'étais présent et
     consentant, et je gardais les vêtements de ceux qui le tuaient.</p>
     <p class="Usuel">21 Et il* me dit&#160;: Va, car je t'enverrai au loin vers les nations.</p>
-    <p class="Note">— ch. 21 v. 31&#160;: <i>ou</i>&#160;: commandant. — ch. 22 v. 1&#160;: <i>tournure hébraïque
-    pour</i>&#160;: Frères et pères. — ch. 22 v. 18, 21&#160;: le, il&#160;: <i>renvoient à</i> «le Juste» <i>du
+    <p class="Note">— v. 31 (ch. 21)&#160;: <i>ou</i>&#160;: commandant. — v. 1 (ch. 22)&#160;: <i>tournure hébraïque
+    pour</i>&#160;: Frères et pères. — v. 18, 21&#160;: le, il&#160;: <i>renvoient à</i> «le Juste» <i>du
     verset</i> 14.</p>
     <p class="Usuel">22 Et ils l'écoutèrent jusqu'à ce mot, et ils élevèrent leur voix, disant&#160;: Ôte de la terre
     un pareil homme, car il n'aurait pas dû vivre.</p>
@@ -63506,7 +63518,7 @@
     <p class="Usuel">21 afin que, comme le péché a régné par la mort, ainsi aussi la grâce régnât par [la] justice pour
     [la] vie éternelle par Jésus Christ notre Seigneur.</p>
     <p class="Note">— v. 12&#160;: <i>la partie doctrinale de l'épître (chap. 1 à 8), jusqu'au chapitre 5:11 traite
-    des</i> péchés<i>&#160;; à partir d'ici, elle s'occupe du</i> péché<i>.</i> — v. 13 à 17&#160;: <i>ces versets
+    des</i> péchés<i>&#160;; à partir d'ici, elle s'occupe du</i> péché<i>.</i> — v. 13&#160;: <i>Les versets 13 à 17
     forment une parenthèse</i>. — v. 14&#160;: <i>comparer</i> Osée 6:7. — v. 15* et 16*&#160;: <i>ou</i>&#160;: il
     n'en est. — v. 15** et 19&#160;: les plusieurs, <i>c'est-à-dire</i> &#160;la masse <i>en contraste avec la personne
     en question</i>. — v. 16**&#160;: <i>c'est-à-dire</i> &#160;était fondé sur une seule chose <i>ou</i> un seul acte,
@@ -63688,9 +63700,9 @@
     <p class="Usuel">32 Celui même qui n'a pas épargné son propre Fils, mais qui l'a livré pour nous tous, comment ne
     nous fera-t-il pas don aussi, librement, de toutes choses avec lui&#160;?</p>
     <p class="Usuel">33 Qui intentera accusation contre des élus de Dieu&#160;?</p>
-    <p class="Usuel">— C'est Dieu qui justifie&#160;; (34) qui est celui qui condamne&#160;?</p>
+    <p class="Usuel">— C'est Dieu qui justifie&#160;;</p><p class="Usuel">34 qui est celui qui condamne&#160;?</p>
     <p class="Usuel">— C'est Christ qui est mort, mais plutôt qui est aussi ressuscité, qui est aussi à la droite de
-    Dieu, qui aussi intercède pour nous&#160;; (35) qui est-ce qui nous séparera de l'amour du Christ&#160;?
+    Dieu, qui aussi intercède pour nous&#160;;</p><p class="Usuel">35 qui est-ce qui nous séparera de l'amour du Christ&#160;?
     Tribulation, ou détresse, ou persécution, ou famine, ou nudité, ou péril, ou épée&#160;?</p>
     <p class="Usuel">36 Selon qu'il est écrit&#160;: «Pour l'amour de toi, nous sommes mis à mort tout le jour&#160;;
     nous avons été estimés comme des brebis de tuerie» <i>[Psaume 44:22]</i>.</p>
@@ -63768,7 +63780,7 @@
     <p class="Usuel">32 Pourquoi&#160;? — Parce que ce n'a point été sur le principe de la foi, mais comme sur le
     principe des œuvres&#160;: car ils ont heurté contre la pierre d'achoppement,</p>
     <p class="Usuel">33 selon qu'il est écrit&#160;: «Voici, je mets en Sion une pierre d'achoppement, et un rocher de
-    chute», et «celui qui croit en lui ne sera pas confus» <i>[Ésaïe 8:14&#160;; 28:16*]</i>.</p>
+    chute», et «celui qui croit en lui ne sera pas confus» <i>[Ésaïe 8:14&#160;; 28:16]</i>*.</p>
     <p class="Note">— v. 33&#160;: <i>citation d'après la version grecque des Septante</i>, <i>comme</i> <i>aussi</i>
     10:11, <i>et souvent</i> <i>ailleurs</i>.</p>
     <p class="Chapitre"><a name="nt06_10" id="nt06_10"></a>Chapitre 10</p>
@@ -64340,7 +64352,7 @@
     <p class="Usuel">15 Car quand vous auriez dix mille maîtres* dans le Christ, vous n'avez cependant pas beaucoup de
     pères, car moi je vous ai engendrés dans le christ Jésus par l'évangile.</p>
     <p class="Usuel">16 Je vous supplie* donc d'être mes imitateurs.</p>
-    <p class="Note">— v. 6&#160;: <i>litt</i>.: l'un pour l'un contre l'autre. — v. 15 <i>litt</i>.: pédagogues,
+    <p class="Note">— v. 6&#160;: <i>litt</i>.: l'un pour l'un contre l'autre. — v. 15&#160;: <i>litt</i>.: pédagogues,
     précepteurs. — v. 16&#160;: <i>ailleurs</i>&#160;: exhorter, encourager.</p>
     <p class="Usuel">17 C'est pourquoi je vous ai envoyé Timothée, qui est mon enfant bien-aimé et qui est fidèle dans
     le Seigneur&#160;; il vous fera souvenir de mes voies en Christ, selon que j'enseigne partout dans chaque
@@ -67561,7 +67573,7 @@
     <p class="Usuel">&#160;</p>
     <p class="Usuel">&#160;</p>
     <p class="Livre"><a name="nt18" id="nt18"></a>Épître à PHILÉMON</p>
-    <p class="Usuel">&#160;</p>
+    <p class="Chapitre"><a name="nt18_01" id="nt18_01"/>Chapitre 1</p>
     <p class="Usuel">1 Paul, prisonnier de Jésus Christ, et le frère Timothée, à Philémon, le bien-aimé et notre
     compagnon d'œuvre,</p>
     <p class="Usuel">2 et à la sœur Apphie, et à Archippe notre compagnon d'armes, et à l'assemblée qui [se réunit]
@@ -68484,7 +68496,7 @@
     avec envie&#160;?</p>
     <p class="Usuel">6 Mais il donne une plus grande grâce. C'est pourquoi il dit&#160;: «Dieu résiste aux orgueilleux,
     mais il donne [la] grâce aux humbles» <i>[Proverbes 3:34]</i>.</p>
-    <p class="Usuel">7 Soumettez-vous donc à Dieu. *Résistez au diable, et il s'enfuira de vous.</p>
+    <p class="Usuel">7 Soumettez-vous donc à Dieu. * Résistez au diable, et il s'enfuira de vous.</p>
     <p class="Usuel">8 Approchez-vous de Dieu, et il s'approchera de vous. Nettoyez vos mains, pécheurs, et purifiez
     vos cœurs, vous qui êtes doubles de cœur*.</p>
     <p class="Usuel">9 Sentez vos misères, et menez deuil et pleurez. Que votre rire se change en deuil, et votre joie
@@ -68915,7 +68927,7 @@
     <p class="Note">— v. 1*&#160;: <i>c'est-à-dire</i> &#160;: écoles <i>ou</i> doctrines (non pas schismes),
     <i>comme</i> en 1 Corinthiens 11:19. — v. 1**&#160;: maître d'un esclave, <i>ailleurs</i> seigneur <i>ou</i>
     souverain&#160;; <i>comme</i> Luc 2:29. — v. 5&#160;: <i>c'est-à-dire</i> &#160;: avec sept autres&#160;; Genèse
-    7:8. — v. 6 (et 6 à 8)&#160;: <i>voir</i> Genèse 19. — v. 8&#160;: actes sans loi, sans frein&#160;; <i>voir</i>
+    7:8. — v. 6&#160;: versets 6 à 8&#160;: <i>voir</i> Genèse 19. — v. 8&#160;: actes sans loi, sans frein&#160;; <i>voir</i>
     Hébreux 8:12, <i>et</i> 1 Corinthiens 9:21. — v. 10&#160;: <i>litt</i>.: marchent après. — v. 11*&#160;:
     n'admettant que leur propre opinion&#160;; <i>voir note à</i> Tite 1:7. — v. 11**&#160;: <i>litt.</i>: gloires. —
     v. 13* et 15&#160;: <i>ailleurs</i> aussi&#160;: injustice&#160;; <i>comme</i> Jacques 3:6. — v. 13**&#160;:
@@ -69214,7 +69226,7 @@
     <p class="Usuel">&#160;</p>
     <p class="Usuel">&#160;</p>
     <p class="Livre"><a name="nt24" id="nt24"></a>Deuxième Épître de JEAN</p>
-    <p class="Usuel">&#160;</p>
+    <p class="Chapitre"><a name="nt24_01" id="nt24_01"/>Chapitre 1</p>
     <p class="Usuel">1 L'ancien à la dame élue et à ses enfants, que j'aime dans [la] vérité, — et non pas moi seul,
     mais aussi tous ceux qui connaissent la vérité,</p>
     <p class="Usuel">2 — à cause de la vérité qui demeure en nous et qui sera avec nous à jamais.</p>
@@ -69244,7 +69256,7 @@
     <p class="Usuel">&#160;</p>
     <p class="Usuel">&#160;</p>
     <p class="Livre"><a name="nt25" id="nt25"></a>Troisième Épître de JEAN</p>
-    <p class="Usuel">&#160;</p>
+    <p class="Chapitre"><a name="nt25_01" id="nt25_01"/>Chapitre 1</p>
     <p class="Usuel">1 L'ancien à Gaïus, le bien-aimé, que j'aime dans [la] vérité.</p>
     <p class="Usuel">&#160;</p>
     <p class="Usuel">2 Bien-aimé, je souhaite qu'à tous égards* tu prospères et que tu sois en bonne santé, comme ton
@@ -69277,7 +69289,7 @@
     <p class="Usuel">&#160;</p>
     <p class="Usuel">&#160;</p>
     <p class="Livre"><a name="nt26" id="nt26"></a>Épître de JUDE</p>
-    <p class="Usuel">&#160;</p>
+    <p class="Chapitre"><a name="nt26_01" id="nt26_01"/>Chapitre 1</p>
     <p class="Usuel">1 Jude, esclave de Jésus Christ et frère de Jacques, aux appelés, bien-aimés en Dieu le Père, et
     conservés en* Jésus Christ&#160;:</p>
     <p class="Usuel">2 Que la miséricorde, et la paix, et l'amour vous soient multipliés&#160;!</p>
@@ -69344,7 +69356,7 @@
     <p class="Usuel">&#160;</p>
     <p class="Usuel">&#160;</p>
     <p class="Livre"><a name="nt27" id="nt27"></a>APOCALYPSE</p>
-    <p class="Note">ou&#160;: RÉVÉLATION</p>
+    <p class="Sous-Titre">*</p><p class="Note">— *&#160;: <i>ou</i>&#160;: RÉVÉLATION</p>
     <p class="Usuel">&#160;</p>
     <p class="AccesDirect">Accès direct aux chapitres de l'Apocalypse&#160;: <a href="#nt27_01">1</a>, <a href=
     "#nt27_02">2</a>, <a href="#nt27_03">3</a>, <a href="#nt27_04">4</a>, <a href="#nt27_05">5</a>, <a href=
@@ -70354,7 +70366,7 @@
     esprits des prophètes, a envoyé son ange, pour montrer à ses esclaves les choses qui doivent arriver bientôt.</p>
     <p class="Usuel">7 Et voici, je viens bientôt*. Bienheureux celui qui garde les paroles de la prophétie de ce
     livre.</p>
-    <p class="Note">— v. 7, 12, 20&#160;: promptement, vitement.</p>
+    <p class="Note">— v. 7&#160;: promptement, vitement.</p>
     <p class="Usuel">8 Et c'est moi, Jean, qui ai entendu et vu ces choses&#160;; et quand j'eus entendu et que j'eus
     vu, je tombai à terre pour rendre hommage devant les pieds de l'ange qui me montrait ces choses.</p>
     <p class="Usuel">9 Et il me dit&#160;: Garde-toi de le faire&#160;; je suis ton compagnon d'esclavage et [celui] de
@@ -70392,7 +70404,6 @@
     <i>omettent</i>&#160;: tous, <i>et ajoutent</i>&#160;: Amen.</p>
     <p class="Usuel">&#160;</p>
     <p class="Usuel">&#160;</p>
-    <p class="Usuel" align="center">&#160;</p>
   </div>
 </body>
 </html>
